### ISSUE TITLE
Loss 及 MAE、MSE 过高的问题

### ISSUE THREAD

**Issue by hhhaiying** [ZPDu/Domain-general-Crowd-Counting-in-Unseen-Scenarios#13]
您好，我在使用代码训练 ShanghaiTech Part A 时，使用默认参数，但发现训时Loss, MAE, 和MSE 很大。  请问您有参数调整建议吗？谢谢！

![Image](https://github.com/user-attachments/assets/e0fe2f13-1a45-45d5-8aec-2e80102781e4)

**Comment by ZPDu** [comment]
您好，请问您有尝试过测试结果是否正确吗？训练过程使用了[label amplification](https://github.com/ZPDu/Domain-general-Crowd-Counting-in-Unseen-Scenarios/blob/20835275c8443066aa51206bc48864aca93c5abd/main.py#L47)将原本的密度图增大了1000倍，所以loss可能会看起来很大。

**Comment by hhhaiying** [comment]
您好，谢谢您的回复！

我的测试结果与预期相差较大，我不太确定具体原因。

![Image](https://github.com/user-attachments/assets/4f73cc76-b724-4ca7-b6a0-ff601ac7012d)

![Image](https://github.com/user-attachments/assets/cb2ed959-2bb6-4ee1-a785-0ae34ce04978)

**Comment by ZPDu** [comment]
您好，我刚才检查了问题出在test.py的模型导入阶段，如果您测试自己的模型，需要把23行改正为net.load_state_dict(checkpoint['state_dict'])。我之后会更正这一部分代码

**Comment by ZPDu** [comment]
test file已经修正

**Comment by hhhaiying** [comment]
收到，太感谢了！
